### DISCUSSION TITLE
clases /sesiones en vivo

### DISCUSSION THREAD

**Discussion by JesusDelAngelLR** [Launch-X-Latam/MisionFrontEnd#75] [General]
todos los dias hay sesion en vivo?? alguien que me ayude

**Comment by manuel260300** [comment]
El mission commander menciono que solo miércoles y viernes a las 7 (hora México) con duración de 1 a 1 1/2 hora

**Comment by iam-leo** [comment]
En Teams, dentro del canal general publicaron el calendario con las sesiones y horarios según los distintos países, revísalo! Siempre serán a la misma hora a menos que avisen de antemano que se cambia el horario del live por algún motivo.
